Restore.cmd

```diff
@@ -7,6 +7,7 @@ set NuGetAdditionalCommandLineArgs=-verbosity quiet -configfile "%RoslynRoot%nug
 :ParseArguments
 if /I "%1" == "/?" goto :Usage
 if /I "%1" == "/clean" set RestoreClean=true&&shift&& goto :ParseArguments
+if /I "%1" == "/fast" set RestoreFast=true&&shift&& goto :ParseArguments
```

Comment:
Why the /fast flag? There's correctness reasons for doing the deletions, it's not just paranoia.